src/messages/de.json

```diff
@@ -40,19 +34,25 @@
   },
   "Error": {
     "notFoundTitle": "Hoppla! Du hast dich verirrt.",
-    "unexpectedTitle": "Es ist ein unerwarteter Fehler aufgetreten!",
     "notFoundSubtitle": "Die gesuchte Seite wurde nicht gefunden.",
-    "genericSubtitle": "Falls dieses Problem weiterhin besteht, melden sie es dem Administrator der Seite.",
-    "returnBtn":"Zurück zur Startseite"
+    "dataEmptyTitle": "Ressource beinhaltet keine Daten!",
+    "dataEmptyMessage": "Endpunkt \"{source}\" für Ressource \"{name}\" mit id {id} hat keine/leere Daten zurückgegeben. Falls dieses Problem weiterhin besteht, melden Sie es dem Administrator der Seite.",
+    "dataNotLoadedTitle": "Daten konnten nicht geladen werden!",
+    "dataNotLoadedMessage": "Die Daten der Ressource \"{name}\" mit Id {id} und Endpunkt \"{source}\" konnten nicht geladen werden. Falls dieses Problem weiterhin besteht, melden Sie es dem Administrator der Seite.\n\nFehler: \"{error}\"",
+    "resourceConfigurationInvalidTitle": "Konfiguration der Ressource ist fehlerhaft!",
+    "resourceConfigurationInvalidMessage": "Die Validierung der Ressource \"{name}\" mit Id {id} und Endpunkt \"{source}\" hat Fehler zurückgegeben. Falls dieses Problem weiterhin besteht, melden Sie es dem Administrator der Seite.\n\nFehler: \"{error}\"",
+    "unexpectedTitle": "Es ist ein unerwarteter Fehler aufgetreten!",
+    "unexpectedMessage": "Es ist ein unerwarteter Fehler aufgetreten!\n\nFehler: \"{error}\"",
+    "genericSubtitle": "Falls dieses Problem weiterhin besteht, melden Sie es dem Administrator der Seite.",
```

Comment:
I am not sure if I like this generic Subtitle also being repeated inside several of the other messages. Maybe we can use string templating inside the ErrorComponent to avoid redundancy inside of the message files.

Reply:
I have actually just removed the subtitle from the error messages as it is shown in the UI anyways. Is that fine by you?

Reply:
Yes that is also okay